wiki/en/Getting-Started.md

```diff
@@ -17,14 +17,18 @@ If you have any questions, please see the [Discussions](https://github.com/jamul
 
 ## Installation
 
-To get Jamulus working with your hardware, please follow the installation guide for your platform:
+To get Jamulus working with your operating system, please follow the installation guide for your platform:
 
 * [Windows](/wiki/Installation-for-Windows)
 * [macOS](/wiki/Installation-for-Macintosh)
 * [Linux](/wiki/Installation-for-Linux)
 
 And for now, close all other applications. It’s best to start simple at first.
 
+## Hardware Setup
+
+If you use external audio hardware, plug that in before you start Jamulus. If you haven't set it up, please see the installation guides mentioned above.
```

Comment:
```suggestion
If you use external audio hardware, plug that in before you start Jamulus. If you haven't configured your hardware, please see the installation guides mentioned above.
```

Reply:
Maybe that’s the problem here:

you might need to configure your hardware (turn up gain,…) and set it up in JACK/ASIO4ALL/… see my proposal

Reply:
Maybe better to add the thing about plugging in hardware to the platform-specific guides (which don't currently say this) and remove it from here to avoid confusion here. I think it's unlikely people with external hardware would not have it plugged in at this stage, and "configuration" is platform specific. This guide is for all platforms.

Reply:
BTW I thought this issue was about the links on this page: https://github.com/jamulussoftware/jamuluswebsite/issues/488 If so, you may as well put them in as part of this PR.

Reply:
Adding it to every page means duplication. The main intention about that was that the most problems seem to occur with wrongly configured hardware.

Reply:
See: https://github.com/jamulussoftware/jamuluswebsite/pull/524 it might also go live right away (Fuzzy matching?)

Reply:
> Adding it to every page means duplication

Not sure I understand. If we say  you should plug in your hardware before running Jamulus here, but don't say it in the section about installing Jamulus earlier, what is the benefit of saying it here for the first time?  Perhaps a minor thing, but seems a little confusing.

Reply:
> See: #524 it might also go live right away (Fuzzy matching?)

Since they're links that don't need translation (would be copied from source segment), if you run the update and then create all target scripts, the new links will appear in translations without having to do anything. Translators could validate them eventually but it's not strictly necessary, other than it "artificially" lowers the % of translated text for threshold purposes.

Reply:
> what is the benefit of saying it here for the first time? 

My original intention here was to hint multiple user groups (somewhat experienced PC user without pro audio knowledge should get a hint at the fact that the gear plays a role; inexperienced user who is already overwhelmed by the fact that he needs to install software -> needs a step by step guide) at one of the probably most challenging UX problems: setting up the audio gear. 

Maybe this needs some more general discussion/thoughts?

Reply:
No that's OK, it's just that if we say "If you haven't configured your hardware... " then that leaves the user to ask themselves if they have configured their hardware or not, and if they think they haven't, they have to go to the installation page they probably just came from where we don't in fact tell them to plug in their hardware (not sure why, maybe because it's obvious?) and that has platform-specific instructions (well, for Windows and Linux anyway). Maybe it's just a minor thing.